### PR TITLE
Prepare Holocene testnet & devnet release

### PR DESCRIPTION
**Pull request by sebastianst**
Follow-up to https://github.com/ethereum-optimism/superchain-registry/pull/677 and  https://github.com/ethereum-optimism/op-geth/pull/429

* Sepolia devnets (only Base & OP): `Fri Nov 15 15:00:00 UTC 2024`, unix ts `1731682800`, this Friday
* Sepolia testnets: `Tue Nov 26 15:00:00 UTC 2024`, unix ts `1732633200`, Tuesday in 2 weeks.

### CODE REVIEW

File: op-node/chaincfg/chains_test.go [hunk 1]
```diff
@@ -89,22 +89,22 @@ var sepoliaCfg = rollup.Config{
...
+	BatchInboxAddress:       common.HexToAddress("0xff00000000000000000000000000000011155420"),
+	DepositContractAddress:  common.HexToAddress("0x16fc5058f25648194471939df75cf27a2fdc48bc"),
+	L1SystemConfigAddress:   common.HexToAddress("0x034edd2a225f7f429a63e0f1d2084b9e0a93b538"),
+	RegolithTime:            u64Ptr(0),
+	CanyonTime:              u64Ptr(1699981200),
+	DeltaTime:               u64Ptr(1703203200),
+	EcotoneTime:             u64Ptr(1708534800),
+	FjordTime:               u64Ptr(1716998400),
+	GraniteTime:             u64Ptr(1723478400),
+	HoloceneTime:            u64Ptr(1732633200),
```
Comment (axelKingsley) [inline op-node/chaincfg/chains_test.go:107]: UTC
2024-11-26 15:00:00

File: op-node/chaincfg/chains_test.go [hunk 2]
```diff
@@ -126,22 +126,22 @@ var sepoliaDev0Cfg = rollup.Config{
...
+	BatchInboxAddress:       common.HexToAddress("0xff00000000000000000000000000000011155421"),
+	DepositContractAddress:  common.HexToAddress("0x76114bd29dFcC7a9892240D317E6c7C2A281Ffc6"),
+	L1SystemConfigAddress:   common.HexToAddress("0xa6b72407e2dc9EBF84b839B69A24C88929cf20F7"),
+	RegolithTime:            u64Ptr(0),
+	CanyonTime:              u64Ptr(0),
+	DeltaTime:               u64Ptr(0),
+	EcotoneTime:             u64Ptr(1706634000),
+	FjordTime:               u64Ptr(1715961600),
+	GraniteTime:             u64Ptr(1723046400),
+	HoloceneTime:            u64Ptr(1731682800),
```
Comment (axelKingsley) [inline op-node/chaincfg/chains_test.go:144]: UTC
2024-11-15 15:00:00

**Review by axelKingsley — APPROVED** [review]
PTAL at the timestamps I pulled from the PR to confirm they are what you intend, thanks :).

File: op-node/chaincfg/chains_test.go (hunk 1 above)
Comment (sebastianst) [reply op-node/chaincfg/chains_test.go:107]: yep that matches, put in the reference SCR PR and excerpt in the description.

**Pull request merged**